diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+1) Fast
+2) Can have "back" button / bookmark, better ui overall.
+3) Data-binding
```

Comment:
👍 
Some other answers might be:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.
1/1